src/controllers/activegate/reconciler/statefulset/container_eec.go

```diff
@@ -0,0 +1,111 @@
+package statefulset
+
+import (
+	"fmt"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/controllers/activegate/internal/consts"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const eecIngestPortName = "eec-http"
+const eecIngestPort = 14599
+
+const activeGateInternalCommunicationPort = 9999
+
+var _ ContainerBuilder = (*ExtensionController)(nil)
```

Comment:
What is this for ? (I'm dumb and I wish to learn 😅 )

Reply:
It's a static check to ensure that `ExtensionController` type implements `ContainerBuilder` interface.

Convenient when implementing/refactoring interfaces because you learn a method's missing at compile time which could otherwise come up in unit tests or runtime (in case of a dynamic cast attempt).

![obraz](https://user-images.githubusercontent.com/16673172/150811651-0cfc109f-4205-4663-9db4-0b380edbbe90.png)